config/zones/PA.yaml

```diff
@@ -4,16 +4,50 @@ bounding_box:
   - - -77.2425664944
     - 9.61161001224
 capacity:
-  biomass: 33
-  coal: 307.86
-  gas: 449.5
-  hydro: 1789
-  hydro storage: 0
-  nuclear: 0
-  oil: 243
-  solar: 522
-  unknown: 0
-  wind: 270
+  biomass:
+    datetime: '2022-01-01'
+    source: IRENA.org
+    value: 68.0
```

Comment:
Tony added a PR that made sure all of these were list items instead of objects so things worked properly in other areas. Could we do the same here?